docs/_docs/reference/enums/enums-index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: index
 title: "Enums"
-nightlyOf: https://docs.scala-lang.org/scala3/reference/enums/index.html
+movedTo: https://docs.scala-lang.org/scala3/reference/enums/index.html
```

Comment:
That should stay as `nightlyOf`

Reply:
I mean, in all the files we should keep `nightlyOf`. This is the mechanism we use to show the banner at the top of every page: 

https://github.com/lampepfl/dotty/blob/04418c3c1256793157e39664a77412b0679916aa/docs/_layouts/static-site-main.html#L7

Reply:
Oh, I see, I really wasn't aware of what was its purpose.